scripts/globals/chocobo.lua

```diff
@@ -0,0 +1,87 @@
+-------------------------------------------------
+--  Author: santssoft
+--  Chocobo functions
+--  Info from:
+--      http://wiki.ffxiclopedia.org/wiki/Chocobo_Renter
+--      http://ffxi.allakhazam.com/wiki/Traveling_in_Vana'diel
+-------------------------------------------------
+
+require("scripts/globals/settings");
+
+--[[-----------------------------------------------
+Description:
+    chocobo = Zone, { [1] Lowest price, [2] Amount added to base price, [3] Amount discounted per time interval, [4] Amount of seconds before price decay, [5] Quest "A Chocobo Riding Game" chance }
+--]]-----------------------------------------------
+
+chocobo = {230,{100,20,5,60,0.10},  -- Southern San d'Oria
```

Comment:
i think this is only referenced in this file, so i am pretty sure it can be a local

also, for readability, you may want to consider using names instead of numbered indexes for tables where the index has no numerical meaning (if you have to say "1 = lowest price", then it's probably better to just call that element "lowestprice" instead of [1]).  if you don't know what i mean (or how to do it) i can give you an example